.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,36 @@
+name: linkcheck
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+env:
+  POETRY_VERSION: "1.3.1"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
```

Comment:
probably only needs to run on one python version

Reply:
Good call.